llvm/include/llvm/IR/ConstantFolder.h

```diff
@@ -20,6 +20,7 @@
 #include "llvm/ADT/STLExtras.h"
 #include "llvm/IR/ConstantFold.h"
 #include "llvm/IR/Constants.h"
+#include "llvm/IR/GEPNoWrapFlags.h"
```

Comment:
Is this include needed? It pulls in IRBuilderFolder.h on the line directly below, so should get it transitively.